# Generated by Django 3.0.8 on 2020-08-17 16:19

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('groups', '0002_group_photo'),
        ('members', '0007_auto_20200815_1148'),
    ]

    operations = [
        migrations.AddField(
            model_name='member',
            name='source',
            field=models.CharField(blank=True, max_length=1, null=True),
        ),
        migrations.CreateModel(
            name='MemberError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('serial', models.CharField(blank=True, max_length=255, null=True)),
                ('name', models.CharField(max_length=256)),
                ('errorfield', models.SlugField(allow_unicode=True)),
                ('description', models.PositiveIntegerField()),
                ('error_date', models.DateTimeField(auto_now=True)),
                ('source', models.CharField(blank=True, max_length=1, null=True)),
                ('group', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='errored_members', to='groups.Group')),
            ],
            options={
                'ordering': ['serial'],
            },
        ),
    ]
